pkg/fanal/secret/builtin-rules.go

```diff
@@ -743,6 +744,24 @@ var builtinRules = []Rule{
 		Regex:    MustCompile(`pscale_tkn_(?i)[a-z0-9\-_\.]{43}`),
 		Keywords: []string{"pscale_tkn_"},
 	},
+	{
+		ID:       "private-packagist-user-token",
+		Category: CategoryPrivatePackagist,
+		Title:    "Private Packagist user token",
+		Severity: "HIGH",
```

Comment:
I couldn't find any documentation on how to choose a severity level for secrets.
How should I determine the level?

Reply:
it's hard to say what seriousness to choose for a particular secret.
Unfortunately, there is no documentation or rules for this
but I think `HIGH` is perfect for this secret